tree/dataframe/src/RLoopManager.cxx

```diff
@@ -97,7 +97,7 @@ static void ExploreBranch(TTree &t, std::set<std::string> &bNamesReg, ColumnName
 
       ExploreBranch(t, bNamesReg, bNames, subBranch, newPrefix, friendName);
 
-      if (t.GetBranch(fullName.c_str())) {
```

Comment:
Can we do `GetBranch() || FindBranch()` to reduce the perf cost?

Reply:
yes! good idea, that would even fix the problems due to `FindBranch` not finding what `GetBranch` can find or `FindBranch` returning a different thing than `GetBranch`.

Reply:
done